app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,64 @@
+class FoodsController < ApplicationController
+  before_action :set_food, only: %i[ show edit update destroy ]
+
+  def index
+    @foods = Food.all
+  end
+
+  def show; end
+
+  def new
+    @food = Food.new
+  end
+
+  def edit; end
+
+  def create
+    @food = Food.new(food_params)
+
+    respond_to do |format|
+      if @food.save
+        format.html { redirect_to @food, notice: "Food was successfully created." }
+        format.json { render :show, status: :created, location: @food }
```

Comment:
Do you need json now? Please remove it for now. Same for all files.
BTW: You can just grep something like `format.json { render` and find all them